cmd/pj-rehearse/main.go

```diff
@@ -146,13 +150,25 @@ func main() {
 
 	changedPresubmits := diffs.GetChangedPresubmits(prowConfig, prowPRConfig, logger)
 
-	success, err := rehearse.ExecuteJobs(changedPresubmits, prNumber, o.candidatePath, jobSpec.Refs, !o.dryRun, logger, pjclient)
+	debugLogger := logrus.New()
+	debugLogger.Level = logrus.DebugLevel
+	if o.debugLogPath != "" {
+		if f, err := os.OpenFile(o.debugLogPath, os.O_APPEND|os.O_WRONLY|os.O_CREATE, os.ModePerm); err == nil {
```

Comment:
is this can be implemented in `Logger.Out` field? It's an `io.Writer`, we can send it to a file directly.

Reply:
Isn't that what I'm doing? `debugLogger` is a `logrus.Logger`, and I'm setting `Out` on line 158.